src/Sylius/Component/Shipping/Checker/ShippingMethodEligibilityChecker.php

```diff
@@ -13,9 +13,16 @@
 
 namespace Sylius\Component\Shipping\Checker;
 
+use const E_USER_DEPRECATED;
+use Sylius\Component\Shipping\Checker\Eligibility\CompositeShippingMethodEligibilityChecker;
 use Sylius\Component\Shipping\Model\ShippingMethodInterface;
 use Sylius\Component\Shipping\Model\ShippingSubjectInterface;
 
+@trigger_error(sprintf('The "%s" class is deprecated since Sylius 1.7, use "%s" instead.', ShippingMethodEligibilityChecker::class, CompositeShippingMethodEligibilityChecker::class), E_USER_DEPRECATED);
```

Comment:
`\E_USER_DEPRECATED` and we don't need a use case

Reply:
What do you mean by `we don't need a use case`?